src/AvaTaxClient.cs

```diff
@@ -284,10 +284,17 @@ private async Task<FileResult> RestCallFileAsync(string verb, AvaTaxPath relativ
 
                 // Read the result
                 if (result.IsSuccessStatusCode) {
+                    string contentType = null;
+                    string filename = null;
+                    if (result.Content.Headers.ContentLength > 0)
+                    {
+                        contentType = result.Content.Headers.GetValues("Content-Type").FirstOrDefault();
+                        GetDispositionFilename(result.Content.Headers.GetValues("Content-Disposition").FirstOrDefault());
```

Comment:
Test is failing filename is null. Just need to add in `filename = ` to the front of this

Reply:
Thanks.  Good catch.